pre-commit.sh

```diff
@@ -105,7 +107,7 @@ function run_integration_tests
     fi
 }
 
-if [[ -z "$PYTHON_VIRTUAL_ENVS" ]]
+if [[ -z "${PYTHON_VIRTUAL_ENVS+1}" ]]
```

Comment:
It would be worth adding a comment here saying what you said in the commit message, including the link to stackexchange.